tooling/generate_instances_types.py

```diff
@@ -0,0 +1,28 @@
+import boto3
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Generate a aws instance type enum.')
+
+parser.add_argument('-r', '--region', help="AWS region, now only works with us-east-1", type=str, default="us-east-1")
```

Comment:
perhaps we should read from environ variables that are available to the AWS CLI

Reply:
@joaoqalves I can set it as a non required parameters and take it from the environment variables if they are not set.

Reply:
done @joaoqalves